### DISCUSSION TITLE
Is it true that local storage encapsulates the state in the component?

### DISCUSSION THREAD

**Discussion by Kazuhiro-Mimaki** [remix-run/remix#7949] [Q&A]
[The remix docs](https://remix.run/docs/en/main/discussion/state-management#local-storage) tells local storage encapsulates the state in the component. It seems to be incorrect to me. What does it mean?
<img width="1470" alt="image" src="https://github.com/remix-run/remix/assets/62820515/29d99d4b-0c48-4bad-8a93-ac7f872fa71e">

**Comment by sergiodxa** [comment]
I don't think they meant that `window.localStorage` does that automatically, but that you can use it to achieve that by using a key that the rest of your app doesn't use.
